build/yamls/antrea-migrator.yml

```diff
@@ -0,0 +1,55 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: antrea
+    component: antrea-migrator
+  name: antrea-migrator
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-migrator
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-migrator
+    spec:
+      hostPID: true
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/os: linux
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - effect: NoSchedule
+          operator: Exists
+        - effect: NoExecute
+          operator: Exists
+      serviceAccountName: antrea-agent
+      volumes:
+        - name: containerd
+          hostPath:
+            path: /run/containerd
+      initContainers:
+        # initContainer will kill all sandboxes container
+        - name: antrea-migrator-init
+          image: antrea/antrea-ubuntu:latest
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          command:
+            - restart_sandbox
```

Comment:
The reason why using `antrea/antrea-ubuntu:latest` is that after deploying Antrea CNI, this image will exist on every Node, so we do not need to pull a new image again. As for the scripts, it will be copied into the image in the Dockerfile by this line. `COPY build/images/scripts/* /usr/local/bin/`